Tests/FreddyTests/JSONSerializingTests.swift

```diff
@@ -17,18 +16,9 @@ class JSONSerializingTests: XCTestCase {
         XCTAssertFalse(string.isEmpty, "There should be characters.")
     }
 
-    func testThatJSONDataIsEqual() {
-        let serializedJSONData = try! json.serialize()
-        let noWhiteSpaceJSON = try! JSON(data: noWhiteSpaceData)
-        let noWhiteSpaceSerializedJSONData = try! noWhiteSpaceJSON.serialize()
-        XCTAssertEqual(serializedJSONData, noWhiteSpaceSerializedJSONData, "Serialized data should be equal.")
-    }
-    
-    func testThatJSONStringIsEqual() {
-        let serializedJSONString = try! json.serializeString()
-        let noWhiteSpaceJSON = try! JSON(data: noWhiteSpaceData)
-        let noWhiteSpaceSerializedJSONString = try! noWhiteSpaceJSON.serializeString()
-        XCTAssertEqual(serializedJSONString, noWhiteSpaceSerializedJSONString, "Serialized string should be equal.")
+    func testThatNoWhitespaceJSONIsEqualToJSON() {
```

Comment:
In Swift 5.0 there's much less likelihood that two unrelated (but otherwise equal) `Dictionary` instances share the same key/value pair ordering. This means when serializing to `Data` (or also as an abstraction to this in Freddy, a `String`), the `Data` and `String` instances returned would not pass equality checks. This is despite their actual _content_ in JSON human-readable terms being equal.

Given there's other tests to ensure two `JSON` instances are equal, the appropriate change here is to check that some `JSON` instance parsed from a fixture is equal to some other `JSON` parsed from a fixture lacking whitespace (but otherwise identical). It can be reasonably assumed that serialized `Data` or `String` representations of any combination of these instances yield contents that whilst not ordered the same are identical for human purposes.